docs/source/dataaccess.md

```diff
@@ -319,15 +359,45 @@ It has no dedicated data store parameters.
 Its common dataset open parameters for opening [xarray.Dataset] instances are
 the same as for the filesystem-based data stores described above.
 
-### ESA SMOS
+### ESA Climate Data Centre `esa-cci-kc`
+
+The data store `esa-cci-kc` accesses datasets that are offered by the [Open Data Portal]
+via the references format.
+
+Data store parameters are the same as for the filesystem-based `reference` store.
+
+Its common dataset open parameters for opening [xarray.Dataset] instances are
+the same as for the filesystem-based data stores described above.
+
+
+### ESA SMOS `smos`
 
-A data store for ESA SMOS data is currently under development 
-and will be released soon.
+The data store `smos` provides L2C datasets of the [ESA Soil Moisture and Ocean Salinity]
+mission.
 
-This data store is provided by the xcube plugin [xcube-smos].
-Once available, you will be able to install it using 
+This data store is provided by the xcube plugin [xcube-smos]. You can install it using
 `conda install -c conda-forge xcube-smos`.
 
+Data store parameters:
+* `source_path: str` - Path or URL into SMOS archive filesystem.
+* `source_protocol: str`: Protocol name for the SMOS archive filesystem.
+* `source_storage_options: dict`: Storage options for the SMOS archive filesystem. See 
+   fsspec documentation for specific filesystems.
+* `cache_path: str`: Path to local cache directory. Must be given, if file caching 
+  is desired. 
+* `xarray_kwargs: dict`: Extra keyword arguments accepted by `xarray.open_dataset`.
+* `extra_source_storage_options`: Extra keyword arguments that override settings in
+  *source_storage_options*.
```

Comment:
```suggestion
* `source_storage_options: dict`: Storage options for the SMOS archive filesystem. See 
   fsspec documentation for specific filesystems. Any option can be overriden by
   passing it as additional data store parameter.
* `cache_path: str`: Path to local cache directory. Must be given, if file caching 
  is desired. 
* `xarray_kwargs: dict`: Extra keyword arguments accepted by `xarray.open_dataset`.
```
I propose this way of documentation to explain that there is not really a keyword argument "extra_source_storage_options".